src/Symfony/Component/Routing/Matcher/Dumper/PhpMatcherDumper.php

```diff
@@ -215,7 +228,7 @@ private function compilePrefixRoutes(DumperPrefixCollection $collection, $suppor
      *
      * @throws \LogicException
      */
-    private function compileRoute(Route $route, $name, $supportsRedirections, $parentPrefix = null)
+    private function compileRoute(Route $route, $name, $supportsRedirections, $ifOrElseIf, $parentPrefix = null)
```

Comment:
Looks like you don't need to add this argument here as it's not used in the method.